libparsec/crates/client/tests/unit/certif/add_user_certificate.rs

```diff
@@ -27,6 +27,27 @@ async fn ok(env: &TestbedEnv) {
     p_assert_matches!(switch, MaybeRedactedSwitch::NoSwitch)
 }
 
+#[parsec_test(testbed = "minimal")]
+async fn multiple(env: &TestbedEnv) {
+    let env = env.customize(|builder| {
+        builder.new_user("bob");
+    });
+    let alice = env.local_device("alice@dev1");
+    let ops = certificates_ops_factory(&env, &alice).await;
+
+    let switch = ops
+        .add_certificates_batch(
+            &env.get_common_certificates_signed(),
+            &[],
+            &[],
+            &Default::default(),
+        )
+        .await
+        .unwrap();
+
+    p_assert_matches!(switch, MaybeRedactedSwitch::NoSwitch)
+}
```

Comment:
How there is multiple certs at play here ?

Reply:
Because there is also `alice` user in `env`

Reply:
add a comment then ;-)

Reply:
Done